pybmix/core/pybmixcpp/example.cpp

```diff
@@ -16,23 +16,31 @@ int add(int i, int j) { return i + j; }
 int subtract(int i, int j) { return i - j; }
 
 int draw_uniform(int start, int end) {
-  Eigen::VectorXd probas = Eigen::VectorXd::Ones(end - start);
-  probas /= probas.sum();
-  return bayesmix::categorical_rng(probas, bayesmix::Rng::Instance().get(),
-                                   start);
+    Eigen::VectorXd probas = Eigen::VectorXd::Ones(end - start);
+    probas /= probas.sum();
+    return bayesmix::categorical_rng(probas, bayesmix::Rng::Instance().get(),
+                                     start);
 }
 
-PYBIND11_MODULE(pybmixcpp, m) {
-  m.def("add", &add);
-  m.def("subtract", &subtract);
-  m.def("draw_uniform", &draw_uniform);
-
-  py::class_<AlgorithmWrapper>(m, "AlgorithmWrapper")
-      .def(py::init<>())
-      .def(py::init<const std::string&, const std::string&, const std::string&,
-                    const std::string&, const std::string&, const std::string&,
-                    const std::string&>())
-      .def("say_hello", &AlgorithmWrapper::say_hello)
-      .def("run", &AlgorithmWrapper::run)
-      .def("get_collector", &AlgorithmWrapper::get_collector);
+PYBIND11_MODULE(pybmixcpp, m
+) {
+m.def("add", &add);
+m.def("subtract", &subtract);
+m.def("draw_uniform", &draw_uniform);
+
+py::class_<AlgorithmWrapper>(m,
+"AlgorithmWrapper")
+.
+
+def (py::init<>())
+
+.
+
+def (py::init<const std::string &, const std::string &, const std::string &,
+        const std::string &, const std::string &, const std::string &,
+        const std::string &>())
+
+.def("say_hello", &AlgorithmWrapper::say_hello)
+.def("run", &AlgorithmWrapper::run)
+.def("get_collector", &AlgorithmWrapper::get_collector);
```

Comment:
something weird is going on here

Reply:
apparently clion went crazy, corrected already